azkaban-common/src/test/java/azkaban/executor/ExecutorManagerTest.java

```diff
@@ -365,6 +389,9 @@ private void testSetUpForRunningFlows() throws Exception {
     //so that flows will be dispatched to executors.
     this.props.put(ExecutorManager.AZKABAN_QUEUEPROCESSING_ENABLED, "true");
 
+    // allow two concurrent runs give one Flow
+    this.props.put("azkaban.max.concurrent.runs.oneflow", 2);
```

Comment:
should it be AZKABAN_MAX_CONCURRENT_RUNS_ONEFLOW instead of the hard coded string?

Reply:
Fixed it!